integration-tests/oidc/src/main/resources/application.properties

```diff
@@ -9,6 +9,18 @@ quarkus.oidc.tls.trust-store-file=client-truststore.jks
 quarkus.oidc.tls.trust-store-password=password
 quarkus.oidc.tls.key-store-file=client-keystore.jks
 quarkus.oidc.tls.key-store-password=password
+
+%tls-registry.quarkus.oidc.tls.tls-configuration-name=oidc-tls
```

Comment:
@michalvavrik Can you clarify please this setup ? Why to declare empty legacy properties ?

Reply:
Because Quarkus config profile `tls-registry` has priority, but properties set without this profile are still effective. Have a look at lines 9, 10, 11. It is not a trick, when I set `=` I am leveling the ground to have same state, as when you don't set it.

Reply:
Maybe I should add additional validation to make old and new properties exclusive. My thinking was that users won't see the old properties and that nothing bad happens if they configure both.